src/type-mappers/object.ts

```diff
@@ -32,21 +32,12 @@ export function extractAdditionalPropertiesType(
   if (swaggerType.type && swaggerType.type !== "object") {
     return undefined;
   }
-  if (swaggerType.additionalProperties === false) {
-    return undefined;
-  }
   if (
-    swaggerType.additionalProperties === undefined ||
-    swaggerType.additionalProperties === true
+    swaggerType.additionalProperties === false ||
```

Comment:
Shouldn't the false-case be moved from this if check to the one on line 35 for `false`?